### PR TITLE
fix: fix ignition icon flickering

### PR DESCRIPTION
**Pull request by RavenInd**
-fix ignition icon flickering (ToolsBlock)

### CODE REVIEW

File: docs/src/components/Sidebar.tsx [hunk 1]
```diff
@@ -88,7 +89,11 @@ const Sidebar = ({ elementsList }: Props) => {
           sidebarItem.href !== undefined &&
           router?.asPath.indexOf(sidebarItem.href) > -1;
         return (
-          <SidebarItem key={sidebarItem.label} className={sidebarItem.type}>
+          <SidebarItem
+            key={sidebarItem.label}
+            className={sidebarItem.type}
+            onClick={() => closeSidebar()}
```
Comment (usulpro) [inline docs/src/components/Sidebar.tsx]:
```suggestion
            onClick={closeSidebar}
```

File: docs/src/components/Sidebar.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { IDocumentationSidebarStructure } from "./types";
 
 interface Props {
   elementsList: IDocumentationSidebarStructure;
+  closeSidebar: () => void;
```
Comment (usulpro) [inline docs/src/components/Sidebar.tsx]:
```suggestion
  closeSidebar?: () => void;
```

File: docs/src/components/Navigation.stories.tsx [hunk 3]
```diff
@@ -24,6 +24,7 @@ export const MobileSidebar = () => (
 
 export const SidebarMenu = () => (
   <Sidebar
+    closeSidebar={() => {}}
```
Comment (usulpro) [inline docs/src/components/Navigation.stories.tsx]:
```suggestion
```

File: docs/src/components/Sidebar.tsx [hunk 4]
```diff
@@ -108,7 +113,8 @@ const Sidebar = ({ elementsList }: Props) => {
 
                   const isAnchor = subItem.href.includes("#");
                   return (
-                    <li key={subItem.label}>
+                    // eslint-disable-next-line
+                    <li key={subItem.label} onClick={() => closeSidebar()}>
```
Comment (usulpro) [inline docs/src/components/Sidebar.tsx]:
```suggestion
                    <li key={subItem.label} onClick={closeSidebar}>
```

File: docs/src/components/landingBlocks/ToolsBlock.styled.tsx [hunk 5]
```diff
@@ -143,16 +143,16 @@ const ButtonContainer = styled.div`
...
     box-shadow: 0 1px 2px ${tm(({ colors }) => colors.sliderButtonHoverShadow)};
 
     &.active {
-      width: 56px;
-      height: 56px;
+      transform-origin: center;
+      transform: scale(1.15);
       box-shadow: 0 1px 2px ${tm(({ colors }) => colors.transparent)};
-      padding: 0;
+      padding: 0px;
```
Comment (usulpro) [inline docs/src/components/landingBlocks/ToolsBlock.styled.tsx:155]: :)

File: docs/src/components/landingBlocks/ToolsBlock.styled.tsx [hunk 6]
```diff
@@ -163,7 +163,8 @@ const ButtonContainer = styled.div`
       &.active {
         width: 72px;
         height: 72px;
-        padding: 0;
+        padding: 0px;
```
Comment (usulpro) [inline docs/src/components/landingBlocks/ToolsBlock.styled.tsx:166]: :)

File: docs/src/components/MobileSidebarMenu.tsx [hunk 7]
```diff
@@ -158,14 +154,15 @@ const MobileSidebarMenu: FC<Props> = ({
             <MenuItem
               key={menuItem.label}
               data-current={router?.asPath === menuItem.href}
+              onClick={() => closeSidebar()}
```
Comment (usulpro) [inline docs/src/components/MobileSidebarMenu.tsx]:
```suggestion
              onClick={closeSidebar}
```

**Review by usulpro — APPROVED** [review]
Please fix small issues as per my comments and go-ahead to merge this

File: docs/src/components/Sidebar.tsx (hunk 1 above)
Comment (RavenInd) [reply docs/src/components/Sidebar.tsx]: corrected

File: docs/src/components/Sidebar.tsx (hunk 2 above)
Comment (RavenInd) [reply docs/src/components/Sidebar.tsx]: done

File: docs/src/components/Navigation.stories.tsx (hunk 3 above)
Comment (RavenInd) [reply docs/src/components/Navigation.stories.tsx]: done

File: docs/src/components/Sidebar.tsx (hunk 4 above)
Comment (RavenInd) [reply docs/src/components/Sidebar.tsx]: corrected

File: docs/src/components/landingBlocks/ToolsBlock.styled.tsx (hunk 5 above)
Comment (RavenInd) [reply docs/src/components/landingBlocks/ToolsBlock.styled.tsx:155]: :)

File: docs/src/components/landingBlocks/ToolsBlock.styled.tsx (hunk 6 above)
Comment (RavenInd) [reply docs/src/components/landingBlocks/ToolsBlock.styled.tsx:166]: :)

File: docs/src/components/MobileSidebarMenu.tsx (hunk 7 above)
Comment (RavenInd) [reply docs/src/components/MobileSidebarMenu.tsx]: Corrected

**Pull request merged**